extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeStreamInfoItemTest.java

```diff
@@ -0,0 +1,83 @@
+package org.schabi.newpipe.extractor.services.youtube;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.schabi.newpipe.downloader.DownloaderFactory.getMockPath;
+
+import com.grack.nanojson.JsonParser;
+import com.grack.nanojson.JsonParserException;
+
+import org.junit.jupiter.api.Test;
+import org.schabi.newpipe.extractor.localization.Localization;
+import org.schabi.newpipe.extractor.localization.TimeAgoPatternsManager;
+import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamInfoItemExtractor;
+import org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamInfoItemLockupExtractor;
+import org.schabi.newpipe.extractor.stream.StreamType;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+
+public class YoutubeStreamInfoItemTest {
+    @Test
+    void videoRendererPremiere() throws FileNotFoundException, JsonParserException {
```

Comment:
It is not a `videoRenderer`, it's a video `lockupViewModel`. This needs to be changed in all relevant places.

Reply:
And as you can see I'm instantiating `YoutubeStreamInfoItemExtractor` instead of `YoutubeStreamInfoItemLockupExtractor`